### ISSUE TITLE
D-009: Low Contrast Text Fails WCAG 2.1 AA

### ISSUE THREAD

**Issue by Keamogetsw3** [Keamogetsw3/CleanCity-Waste_Pickup_Scheduler-QATesting#56]
## Steps to Reproduce
- Open the application in Chrome.
- Open DevTools → Lighthouse.
- Run an Accessibility audit.
- Observe the Contrast section showing failing elements.

## Expected Result
All text and UI elements should meet or exceed WCAG 2.1 AA requirements.

## Actual Results
Low Contrast Text Fails WCAG 2.1 AA

## Evidence:

<img width="1366" height="768" alt="Image" src="https://github.com/user-attachments/assets/9b198eb4-065f-4a1e-8bdf-f99c4387115a" />
